### PR TITLE
Modify link context on Storage Classes page

### PR DESCRIPTION
**Pull request by kadtendulkar**
Follow up on #36079

Updated the page [Storage Classes](https://kubernetes.io/docs/concepts/storage/storage-classes/)

### CODE REVIEW

**Review by bishal7679 — COMMENTED** [review]
/approve
/lgtm
Thank you @kadtendulkar

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by bishal7679 — APPROVED** [review]
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

File: content/en/docs/concepts/storage/storage-classes.md [hunk 1]
```diff
@@ -413,7 +413,7 @@ parameters:
     * Distribute volume: `volumetype: none`
 
     For available volume types and administration options, refer to the
-    [Administration Guide](https://access.redhat.com/documentation/en-US/Red_Hat_Storage/3.1/html/Administration_Guide/part-Overview.html).
+    [Administration Guide](https://access.redhat.com/documentation/en-US/Red_Hat_Storage/3.1).
```
Comment (reylejano) [inline content/en/docs/concepts/storage/storage-classes.md:416]: I think this is a better link as it does not lock into a specific version and forwards to the latest version, atm is 3.5
```suggestion
    [Administration Guide](https://access.redhat.com/documentation/en-us/red_hat_gluster_storage/).
```

Comment (kundan2707) [reply content/en/docs/concepts/storage/storage-classes.md:416]: Suggestion given by @reylejano  looks better

**Comment by reylejano** [comment]
Thank you for addressing feedback
/approve

**Review by kundan2707 — COMMENTED** [review]
ok now

**Review by ydFu — APPROVED** [review]
/lgtm

thank for fix it.

**Comment by mehabhalodiya** [comment]
Closing this per https://github.com/kubernetes/website/pull/36196#issuecomment-1224289897

/close

**Pull request closed without merging**